backend/api/swagger/run.swagger.json

```diff
@@ -48,14 +49,14 @@
           },
           {
             "name": "sort_by",
-            "description": "Can be format of \"field_name\", \"field_name asc\" or \"field_name des\"\nAscending by default.",
+            "description": "Options include \"field_name\", \"field_name asc\", or \"field_name des\". (Example, \"name asc\" or \"id des\")\nAscending by default.",
             "in": "query",
             "required": false,
             "type": "string"
           },
           {
-            "name": "resource_reference_key.type",
-            "description": "The type of the resource that referred to.",
+            "name": "resources_reference_key.type",
```

Comment:
I feel resource_reference_key is currently used in order to be consistent with the naming of this input in backend api definition https://github.com/kubeflow/pipelines/blob/a9602fbc3fca353408433656adbcdb4c6f418585/backend/api/run.proto#L148